README.md

```diff
@@ -70,3 +77,9 @@ yarn test:e2e
 ```bash
 yarn test
 ```
+
+## Deployments
+
+- http://app.opendollar.com/
+- dev branch http://app.optimism-goerli.opendollar.com/
```

Comment:
We're not using optimism or optimism/arbitrum goerli anymore

Reply:
I will take a look on the latest url we use. Thanks @jahabeebs !